src/core/server/graph/schema/schema.graphql

```diff
@@ -1240,6 +1240,22 @@ type StoryMessageBox {
   content: String
 }
 
+################################################################################
+## Embed Links
+################################################################################
+
+type EmbedLinksConfiguration {
```

Comment:
As this can likely be the integration point for other embedded content, suggest renaming:

```suggestion
type EmbedConfiguration {
```